cryptofeed/exchange/huobi_dm.py

```diff
@@ -0,0 +1,146 @@
+'''
+Huobi_DM has 3 futures per currency (with USD as base): weekly, bi-weekly(the next week), and quarterly.
+
+You must subscribe to them with: CRY_TC
+where
+   CRY = BTC, ETC, etc.
+   TC is the time code mapping below:
+     mapping  = {
+         "this_week": "CW", # current week
+         "next_week": "NW", # next week
+         "quarter": "CQ"    # current quarter
+     }
+
+So for example, to get the quarterly BTC future, you subscribe to "BTC_CQ", and it is returned to channel "market.BTC_CQ. ..."
+However since the actual contract changes over time, we want to pubish the pair name using the actual expiry date, which
+is contained in the exchanges "contract_code".
+
+Here's what you get for BTC querying https://www.hbdm.com/api/v1/contract_contract_info on 2019 Aug 16:
+[{"symbol":"BTC","contract_code":"BTC190816","contract_type":"this_week","contract_size":100.000000000000000000,"price_tick":0.010000000000000000,"delivery_date":"20190816","create_date":"20190802","contract_status":1}
+,{"symbol":"BTC","contract_code":"BTC190823","contract_type":"next_week","contract_size":100.000000000000000000,"price_tick":0.010000000000000000,"delivery_date":"20190823","create_date":"20190809","contract_status":1}
+,{"symbol":"BTC","contract_code":"BTC190927","contract_type":"quarter","contract_size":100.000000000000000000,"price_tick":0.010000000000000000,"delivery_date":"20190927","create_date":"20190614","contract_status":1},
+...]
+So we return BTC190927 as the pair name for the BTC quaterly future.
+
+'''
+import logging
+import json
+from decimal import Decimal
+import zlib
+
+from sortedcontainers import SortedDict as sd
+
+from cryptofeed.defines import HUOBI_DM, BUY, SELL, TRADES, BID, ASK, L2_BOOK
+from cryptofeed.exchange.huobi import Huobi
+from cryptofeed.standards import pair_std_to_exchange, pair_exchange_to_std, timestamp_normalize
+
+
+LOG = logging.getLogger('feedhandler')
+
+
+class HuobiDM(Huobi):
```

Comment:
it doesnt look like this class shares anything with the parent class (Huobi). Shouldnt this just inherit from `Feed`?

Reply:
You are right. I originally used parent class, but then after testing, wound up changing every function. Looks like you already changed this. Thanks.